### ISSUE TITLE
Password security

### ISSUE THREAD

**Issue by BarbzYHOOL** [ShahriyarR/MySQL-AutoXtraBackup#306]
Is it safe to have the password written in clear in the configuration file?

Even more worrying, when I use verbose debug mode to see what happens, it shows the password.

And if I log it to a file, it's now also in that log file!

**Comment by ShahriyarR** [comment]
* Yes it is OK to have password in config file as clear text -> it is up to user to secure its file in Linux from the 
* It was fixed and in new version of our tool the output will be like:
```
018-08-01 12:18:25 DEBUG    The following backup command will be executed /usr/bin/xtrabackup --defaults-file=/etc/mysql/mysql.conf.d/mysqld.cnf --user=root --password='*'  --target-dir=/home/shako/XB_TEST/backup_dir/full/2018-08-01_12-18-25 --backup --host=127.0.0.1 --port=3306 --no-version-check
```

**Comment by BarbzYHOOL** [comment]
Ok cool, when is the new version out? :)

**Comment by ShahriyarR** [comment]
Soon :)

**Comment by ShahriyarR** [comment]
Released as 1.5.3.
Closing.
